src/Operator/OperatorInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Operator;
+
+use Twig\Node\Expression\AbstractExpression;
+use Twig\OperatorPrecedenceChange;
+
+interface OperatorInterface
+{
+    public function getOperator(): string;
+
+    /**
+     * @return class-string<AbstractExpression>
```

Comment:
```suggestion
     * @return class-string<AbstractExpression>|null
```

Alternatively, as I see that you throw a LogicException anyway when the operator does not have a node class, maybe the return type could stay non-nullable, by moving the exception in the BC layer instead.